setup.py

```diff
@@ -46,17 +46,21 @@
       package_data={
         'cortx': ['py.typed'],
         'cortx.src.setup': ['*.py'],
+        'cortx.src.support': ['*.py'],
       },
       long_description=long_description,
       zip_safe=False,
       python_requires='>=3.6',
       entry_points={
           'console_scripts': [
               'rgw_setup = cortx.rgw.setup.rgw_setup:main',
+              'rgw_support_bundle = cortx.rgw.support.rgw_support_bundle:main',
               ]
       },
       data_files =[ ('%s/mini-provisioner' % RGW_INSTALL_PATH, mini_prov_files),
                     ('%s/bin' % RGW_INSTALL_PATH,['src/setup/rgw_setup']),
+                    ('%s/bin' % RGW_INSTALL_PATH,['src/support/rgw_support_bundle']),
```

Comment:
Lets merge these two lines into one. rgw_setup and rgw_support_bundle both can be mentioned in a same list.

Reply:
Ack